internal/nfs/controller/volume.go

```diff
@@ -274,7 +281,7 @@ func (nv *NFSVolume) getNFSCluster() (string, error) {
 
 	clusterName, err := j.FetchAttribute(nv.ctx, mdPool, nv.objectUUID, clusterNameKey)
 	if err != nil {
-		return "", fmt.Errorf("failed to get cluster name: %w", err)
+		return "", fmt.Errorf("cluster name for %q %w: %v", nv.objectUUID, ErrNotFound, err)
```

Comment:
This wont be true in all causes we should return `ErrNotFound` only if `FetchAttribute` is returning `ErrKeyNotFound` or `ErrPoolNotFound`